<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vendedores_ui</class>
 <widget class="QWidget" name="vendedores_ui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QMenuBar" name="menuBar">
     <widget class="QMenu" name="menuTrocar_Visualiza_o">
      <property name="title">
       <string>Trocar Visualização</string>
      </property>
      <addaction name="actionLotes_Vendedores"/>
      <addaction name="actionClientes_Vendedores"/>
      <addaction name="actionVendedores_Vendedores"/>
      <addaction name="actionVendas_Vendedores"/>
     </widget>
     <addaction name="menuTrocar_Visualiza_o"/>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,0">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="Menu_Title">
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>32</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>VENDEDORES</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="APP" stretch="1,6">
         <item>
          <layout class="QHBoxLayout" name="FORM">
           <property name="spacing">
            <number>6</number>
           </property>
           <property name="topMargin">
            <number>20</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>20</number>
           </property>
           <item>
            <layout class="QVBoxLayout" name="nome_layout">
             <property name="leftMargin">
              <number>10</number>
             </property>
             <property name="topMargin">
              <number>10</number>
             </property>
             <property name="rightMargin">
              <number>10</number>
             </property>
             <property name="bottomMargin">
              <number>10</number>
             </property>
             <item>
              <widget class="QLabel" name="nome_label">
               <property name="text">
                <string>NOME</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPlainTextEdit" name="nome_form">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>50</height>
                </size>
               </property>
               <property name="tabChangesFocus">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="endereco_layout">
             <property name="leftMargin">
              <number>10</number>
             </property>
             <property name="topMargin">
              <number>10</number>
             </property>
             <property name="rightMargin">
              <number>10</number>
             </property>
             <property name="bottomMargin">
              <number>10</number>
             </property>
             <item>
              <widget class="QLabel" name="endereco_label">
               <property name="text">
                <string>ENDEREÇO</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPlainTextEdit" name="endereco_form">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>50</height>
                </size>
               </property>
               <property name="tabChangesFocus">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="telefone_layout">
             <property name="leftMargin">
              <number>10</number>
             </property>
             <property name="topMargin">
              <number>10</number>
             </property>
             <property name="rightMargin">
              <number>10</number>
             </property>
             <property name="bottomMargin">
              <number>10</number>
             </property>
             <item>
              <widget class="QLabel" name="telefone_label">
               <property name="text">
                <string>TELEFONE</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPlainTextEdit" name="telefone_form">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>50</height>
                </size>
               </property>
               <property name="tabChangesFocus">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="salario_layout">
             <property name="leftMargin">
              <number>10</number>
             </property>
             <property name="topMargin">
              <number>10</number>
             </property>
             <property name="rightMargin">
              <number>10</number>
             </property>
             <property name="bottomMargin">
              <number>10</number>
             </property>
             <item>
              <widget class="QLabel" name="salario_label">
               <property name="text">
                <string>SALÁRIO</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPlainTextEdit" name="salario_form">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>50</height>
                </size>
               </property>
               <property name="tabChangesFocus">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="percentual_layout">
             <property name="leftMargin">
              <number>10</number>
             </property>
             <property name="topMargin">
              <number>10</number>
             </property>
             <property name="rightMargin">
              <number>10</number>
             </property>
             <property name="bottomMargin">
              <number>10</number>
             </property>
             <item>
              <widget class="QLabel" name="percentual_label">
               <property name="text">
                <string>PERCENTUAL DE GANHO</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="percentual_spinBox">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>100</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QVBoxLayout" name="btns_layout">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="add_btn">
               <property name="text">
                <string>ADICIONAR</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="rm_btn">
               <property name="text">
                <string>REMOVER</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="edit_btn">
               <property name="text">
                <string>EDITAR</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QListWidget" name="ListWidget"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionLotes_Vendedores">
   <property name="text">
    <string>Lotes</string>
   </property>
  </action>
  <action name="actionClientes_Vendedores">
   <property name="text">
    <string>Clientes</string>
   </property>
  </action>
  <action name="actionVendas_Vendedores">
   <property name="text">
    <string>Vendas</string>
   </property>
  </action>
  <action name="actionVendedores_Vendedores">
   <property name="text">
    <string>Vendedores</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
